test/unit/TransmuteToken.spec.js

```diff
@@ -37,14 +45,14 @@ contract('TST', (accounts) => {
     assert(tokenAmount - approvedAmount === (await tst.balanceOf(accounts[1])).toNumber());
   });
 
-  it('cannot transfer tokens without approval', async () => {
+  it('should fail to transfer tokens without approval', async () => {
     await assertFail(
       tst.transferFrom(owner, accounts[2], tokenAmount / 10, {from: owner}),
       'should not be able to transfer tokens without approval'
     );
   });
 
-  it('cannot transfer more tokens than approved', async () => {
+  it('should fail to transfer more tokens than approved', async () => {
```

Comment:
technically this is "cannot approve transfer of tokens exceeding user balance" right?